### DISCUSSION TITLE
Embed a URL error on the thumbnail

### DISCUSSION THREAD

**Discussion by utamayo** [GitbookIO/community#815] [0. General bugs and questions]
Many times when I add a embed URL the thumbnail image is the default error image:
![image](https://github.com/user-attachments/assets/ebb695b3-6b05-4c20-9361-a94209c78026)
URLs with thumbnail errors:
https://m2.material.io/
https://m3.material.io/
https://romannurik.github.io/AndroidAssetStudio/index.html
https://www.appicon.co/
https://java-design-patterns.com/patterns/

Any suggestion why is this happening?

**Comment by DanielSmithGK** [comment]
Thanks for raising this, I have the same issue, e.g. with https://app.powerbi.com.
